### PR TITLE
CAPZ: Release v30.0.0.

### PR DESCRIPTION
**Pull request by Gacko**
### Checklist

- [x] Roadmap issue created
- [x] Release uses latest stable Flatcar
- [x] Release uses latest Kubernetes patch version

### Triggering E2E tests

To trigger the E2E test for each new Release added in this PR, add a comment with the following:

`/run releases-test-suites PREVIOUS_RELEASE=29.0.0`

If you want to trigger conformance tests, you can do so by adding a comment similar to the following:

`/run conformance-tests PROVIDER=azure RELEASE_VERSION=30.0.0`

For more details see the [README.md](/README.md#running-tests-against-prs).

### CODE REVIEW

File: azure/v30.0.0/README.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+- Apps: Use `catalog` from Release CR.
+
+### Apps
+
+- azure-cloud-controller-manager from v1.29.8-gs1 to v1.30.6-gs1
+- azure-cloud-node-manager from v1.29.8-gs1 to v1.30.6-gs1
+- cert-exporter from v2.9.1 to v2.9.2
+- cilium from v0.25.1 to v0.27.0
+- node-exporter from v1.19.0 to v1.20.0
+- observability-bundle from v1.5.3 to v1.6.1
```
Comment (TheoBrigitte) [inline azure/v30.0.0/README.md:24]:
```suggestion
- observability-bundle from v1.5.3 to v1.7.0
```

File: azure/v30.0.0/README.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+  - Hubble Relay
+  - Cilium Operator
+
+### node-exporter [v1.19.0...v1.20.0](https://github.com/giantswarm/node-exporter-app/compare/v1.19.0...v1.20.0)
+
+#### Changed
+
+- Synced with upstream chart v4.38.0 (node-exporter 1.8.2).
+
+### observability-bundle [v1.5.3...v1.6.1](https://github.com/giantswarm/observability-bundle/compare/v1.5.3...v1.6.1)
```
Comment (TheoBrigitte) [inline azure/v30.0.0/README.md:64]:
```suggestion
### observability-bundle [v1.5.3...v1.7.0](https://github.com/giantswarm/observability-bundle/compare/v1.5.3...v1.7.0)
```

File: azure/v30.0.0/release.diff [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+    catalog: cluster							    catalog: cluster
+								>           version: 0.1.1
+    dependsOn:								    dependsOn:
+    - cilium								    - cilium
+  - name: node-exporter							  - name: node-exporter
+    version: 1.19.0						|           version: 1.20.0
+    dependsOn:								    dependsOn:
+    - kyverno-crds							    - kyverno-crds
+  - name: observability-bundle						  - name: observability-bundle
+    version: 1.5.3						|           version: 1.6.1
```
Comment (TheoBrigitte) [inline azure/v30.0.0/release.diff:84]:
```suggestion
    version: 1.5.3						|           version: 1.7.0
```

File: azure/v30.0.0/release.yaml [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+    catalog: cluster
+    version: 0.1.1
+    dependsOn:
+    - cilium
+  - name: node-exporter
+    version: 1.20.0
+    dependsOn:
+    - kyverno-crds
+  - name: observability-bundle
+    version: 1.6.1
```
Comment (TheoBrigitte) [inline azure/v30.0.0/release.yaml:83]:
```suggestion
    version: 1.7.0
```

**Review by TheoBrigitte — COMMENTED** [review]
Just a reminder to set observability-bundle to v1.7.0

**Comment by Gacko** [comment]
v30.0.0 is currently not work in progress as we are focussing on other topics right now. As soon as we come back here, we will first update each and every app in this draft.

Feel free to file your requests for a more recent app version in the according `requests.yaml`, so new releases can consider it.

File: azure/v30.0.0/release.yaml [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+  - name: cert-manager
+    version: 3.8.1
+    dependsOn:
+    - prometheus-operator-crd
+  - name: chart-operator-extensions
+    version: 1.1.2
+    dependsOn:
+    - prometheus-operator-crd
+  - name: cilium
+    version: 0.27.0
```
Comment (ubergesundheit) [inline azure/v30.0.0/release.yaml:40]:
```suggestion
    version: 0.29.0
```

**Pull request closed without merging**